arch/sim/src/sim/win/up_hostmisc.c

```diff
@@ -40,6 +48,12 @@
 
 void host_abort(int status)
 {
+#ifdef CONFIG_ARCH_COVERAGE
+  /* Dump gcov data. */
+
+  __gcov_dump();
```

Comment:
let's revert the change. __gcov_dump is specific to gcc/clang, so it can't work with msvc.

Reply:
In this case `CONFIG_ARCH_COVERAGE` shouldn't be enabled, right?

Maybe we should fix the dependencies of `CONFIG_ARCH_COVERAGE`?

Reply:
could be

Reply:
Oh, you mean specifically for Windows? And keeping only the Posix one?

Reply:
Yes.